core/services/chainlink/application.go

```diff
@@ -285,14 +294,40 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	} else {
 		globalLogger.Debug("Off-chain reporting disabled")
 	}
+	terraORM := terra.NewORM(db, globalLogger, cfg)
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
+		// TODO: parameterize poll period
+		nodes, _, err := terraORM.Nodes(0, 1)
```

Comment:
This section of code IMO now pollutes the high-level `application.go` with low-level details about the Terra integration, which should/was contained in the OCR2 delegate before. The delegate takes in high-level dependencies and constructs specific relayers.

Long-term relayers will be multi-protocol (not just OCR2 feeds), so probably the specific OCR2 delegate is also not the place to create the relayers.

Please at least add inline comments that in the near future we'd love to contain these relay construction somewhere else, and keep `application.go` more high-level.

Reply:
This is very much still WIP. See https://github.com/smartcontractkit/chainlink-terra/pull/29 and https://github.com/smartcontractkit/chainlink/pull/5797